source/develop/developer-guide/vdsm/block-storage-domains.html.md

```diff
@@ -10,11 +10,11 @@ For block domains we use LVM for logical volume management inside the domain. We
 
 *   The VG must have it's metadata only on 1 PV. This limitation is because even when LVM reads the metadata, if it will notice discrepancy between the MD on 2 PVs it will try and reconcile it. This could cause metadata corruption when commands are being run on two hosts.
 *   You must allocate at least 100MB of metadata space. VDSM boasts support for 1000s of images, this cannot happen if there is not enough room on the PV for all the volume metadata.
-*   We allocate 5 "special" volumes:
-    1.  metadata - contains the domain\\pool and volume metadata
+*   We allocate 6 "special" volumes:
+    1.  metadata - contains volume metadata
     2.  inbox and outbox - used for the [mailbox](mailbox)
```

Comment:
when in it, could you please remove link to `mailbox`? This page doesn't seems to exist.